stackstac/rio_reader.py

```diff
@@ -435,7 +438,6 @@ def __getstate__(
             "resampling": self.resampling,
             "dtype": self.dtype,
             "fill_value": self.fill_value,
-            "rescale": self.rescale,
```

Comment:
Oh, we definitely need to store `scale_offset` in the pickled state! Otherwise, that will cause some very confusing errors.

Reply:
Thanks for the insight!